### ISSUE TITLE
Investigate unknown packet types

### ISSUE THREAD

**Issue by Monstrofil** [Monstrofil/replays_unpack#18]
E.g. for wows:

```
TIME: 0.0 TYPE: 0x16 SIZE: 12 DATA: <_io.BytesIO object at 0x000001ADFF4BABE8> 22
TIME: 0.0 TYPE: 0x10 SIZE: 1 DATA: <_io.BytesIO object at 0x000001AD809DCB88> 16
TIME: 0.0 TYPE: 0xf SIZE: 8 DATA: <_io.BytesIO object at 0x000001AD809DCE28> 15
TIME: 0.0 TYPE: 0x26 SIZE: 4 DATA: <_io.BytesIO object at 0x000001AD809FA108> 38
TIME: 0.0 TYPE: 0x25 SIZE: 10 DATA: <_io.BytesIO object at 0x000001AD809DCD68> 37
TIME: 0.0 TYPE: 0x10 SIZE: 1 DATA: <_io.BytesIO object at 0x000001AD809DCF48> 16
TIME: 0.0 TYPE: 0xe SIZE: 8 DATA: <_io.BytesIO object at 0x000001AD809FADC8> 14
TIME: 0.0 TYPE: 0x31 SIZE: 8 DATA: <_io.BytesIO object at 0x000001AD809FABE8> 49
TIME: 0.0 TYPE: 0x13 SIZE: 0 DATA: <_io.BytesIO object at 0x000001AD8095A5E8> 19
TIME: 0.3901594877243042 TYPE: 0x24 SIZE: 56 DATA: <_io.BytesIO object at 0x000001AD80A265E8> 36

TIME: 0.0 TYPE: 0xffffffff SIZE: 16 DATA: <_io.BytesIO object at 0x000001AD80A4A6A8> 4294967295
```

The last one is definitely EndGame or something like that,

**Comment by razaqq** [comment]
0x10 seems to be a single bool, no idea what it means
0x16 is a version packet like this
```c++
struct VersionPacket
{
    uint32_t size;
	std::string version;
};
```
0x20 and 0x26 are either 0 or 4 bytes, maybe uint32_t, again no idea what it represents

These two occur hundreds of times, but their payload never changes:
0xE is always 8 bytes
0x18 is always 52 bytes, mostly zeroes
0x2E always single byte, bool
0x2F always 12 bytes, Vec3?
0x29 always 32 bytes
0x25 occurs only once, 10 bytes
0xFFFFFFFF also occurs only once, 16 bytes

**Comment by razaqq** [comment]
You might want to have a look here: https://github.com/lkolbly/wows-replays/issues/14
There is a some more info on other packets there

**Comment by Monstrofil** [comment]
> I'm not sure if this is already known, but `0x16` is extremely useful for World of Tanks because it can tell us when the battle actually starts. This is because the time inside the replay (top right corner) does not reflect the timestamp of a packet when it occurs. With this packet we can calculate the actual replay time of the packet and this can help with decoding packets if we watch the replay along with it.

I don't think this packet exist in wows. The timing data in wows is handled by extra entity called BattleLogic, which has information about time and battle state.

The `0x16` seems to be used for version data:
    b'\x08\x00\x00\x000,11,3,0'

**Comment by razaqq** [comment]
yeah `0x16` for wows is explained in detail above
